.github/workflows/scheduled.yaml

```diff
@@ -56,6 +56,14 @@ jobs:
       )
     name: revdepcheck ↩️
     uses: insightsengineering/r.pkg.template/.github/workflows/revdepcheck.yaml@main
+    with:
+      lookup-refs: |
+        insightsengineering/teal.data
+        insightsengineering/teal.slice
+        insightsengineering/teal.code
+        insightsengineering/teal.logger
+        insightsengineering/teal.reporter
+        insightsengineering/teal.widgets
```

Comment:
@pawelru I added this because `revdepcheck` workflow can handle such input. 
No idea how it worked before without `lookup-refs` in here

https://github.com/insightsengineering/r.pkg.template/blob/main/.github/workflows/revdepcheck.yaml#L96

Reply:
well, it wasn't working :P https://github.com/insightsengineering/teal/actions/runs/11089446447/job/30810578966

Reply:
haha ok : )